src/views/collectable/components/NftData.vue

```diff
@@ -94,50 +93,38 @@ export default {
   setup(props) {
     const { darkMode } = useDarkMode();
 
+    const collectable = ref(props.collectable);
+
+    const ipfsUri = ref(null);
+    const ipfsMediaUrl = ref(null);
     const displayModal = ref(false);
-    emitter.on('openNftDataModal', payload => {
-      displayModal.value = true;
+    
+    const { item: ipfsItem, loading } = useIPFS(ipfsUri);
+
+    watchEffect(async () => {
+      console.log(displayModal.value, collectable.value?.nft_contract_address, collectable.value?.nft_token_id);
```

Comment:
Might be here accidentally but I also don't really mind if it gets logged out

Reply:
removed it